BLAST_loop.py

```diff
@@ -16,9 +16,11 @@
 from Bio.Blast.Applications import NcbiblastnCommandline
 #help(NcbiblastnCommandline)
 import os
+import sys
+directory = sys.argv[-1]	#first argument input after 'python BLAST_loop.py' in command line will be stored in variable 'directory'
```

Comment:
@QuLogic thanks, good catch. @sina-sed, would you be able to modify this? We can chat tomorrow as well.